### ISSUE TITLE
Browser not supported "This browser or app may not be secure"

### ISSUE THREAD

**Issue by TiagoSeverino** [fawazahmed0/youtube-uploader#158]
Password input not showing because google doesn't support this browser

Error: TimeoutError: waiting for selector `input[type="password"]:not([aria-hidden="true"])` failed: timeout 60000ms exceeded
![image](https://user-images.githubusercontent.com/14254842/186283379-e77bdc9d-5713-4183-8313-b854e04b0b17.png)

**Comment by pierreminiggio** [comment]
Maybe related to https://github.com/fawazahmed0/youtube-uploader/issues/71 ?

**Comment by TiagoSeverino** [comment]
> Maybe related to #71 ?

Yeah, it's the same issue, but on my local computer.
I tried in a new project and worked good, might be an mistake on my end.
